common/Linux/LnxMisc.cpp

```diff
@@ -109,9 +109,8 @@ bool ChangeScreenSaverStateDBus(const bool inhibit_requested, const char* progra
 	else
 	{
 		// Only Append the cookie.
-		if (!dbus_message_iter_append_basic(&message_itr, DBUS_TYPE_UINT32, &s_screensaver_dbus_cookie))
+		if (!dbus_message_iter_append_basic(&message_itr, DBUS_TYPE_UINT32, &s_cookie))
 			goto cleanup;
-		s_screensaver_dbus_cookie = 0;
```

Comment:
Intentional, btw, was completely useless.

it was meant for a function that I don't have implemented here because it's unneeded, and even then it was at the wrong position, this is where it should have been: https://github.com/CharlesThobe/screensaver-dbus/blob/master/src/screensaver-dbus.cpp#L50
but since checking for current screensaver status is already implemented in pcsx2 (by setting it each time `UpdateInhibitScreensaver` is called, then all of that is unnecessary.